src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Extensions/IdentityExtensions.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace SFA.DAS.ProviderCommitments.Web.Extensions
 {
-    public static class IIdentityExtensions
+    public static class IdentityExtensions
     {
         public static string Upn(this IIdentity identity)
```

Comment:
if the class is renamed, shouldn't the argument type also be renamed?

Reply:
Reverted